reference/7.1/Microsoft.PowerShell.Utility/Get-FormatData.md

```diff
@@ -149,11 +149,6 @@ TypeNames                               FormatViewDefinition
 {Microsoft.Powershell.Utility.FileHash} {Microsoft.Powershell.Utility.FileHash}
 ```
 
-> [!IMPORTANT]
```

Comment:
I'm not sure why you removed this. Can you explain?

Reply:
In PowerShell 7.1, it will return all formatting by default. If i'm reading things correctly, you should be able to omit the PowerShell version now and receive all appropriate Formatting information. 

https://github.com/PowerShell/PowerShell/issues/4237

Reply:
OK, I was wondering if that got fixed. That makes sense.